packages/form-state-manager/src/types/form-spy.d.ts

```diff
@@ -0,0 +1,11 @@
+import { Subscription } from './use-subscription';
+import AnyObject from './any-object';
+
+type Children = (props: AnyObject) => React.ReactElement;
```

Comment:
I think that we should pass specific props to the children's function. Props like value, and field metadata